SimG4CMS/PPS/interface/PPSDiamondG4Hit.h

```diff
@@ -0,0 +1,122 @@
+///////////////////////////////////////////////////////////////////////////////
+// Author
+// Seyed Mohsen Etesami setesami@cern.ch
+// Feb 2016
+///////////////////////////////////////////////////////////////////////////////
+#ifndef PPS_PPSDiamondG4Hit_h
+#define PPS_PPSDiamondG4Hit_h
+
+#include "G4VHit.hh"
+#include "G4ThreeVector.hh"
+#include <boost/cstdint.hpp>
+#include <iostream>
+
+class PPSDiamondG4Hit : public G4VHit {
+public:
+  PPSDiamondG4Hit();
+  ~PPSDiamondG4Hit() override;
+  PPSDiamondG4Hit(const PPSDiamondG4Hit &right);
+  const PPSDiamondG4Hit &operator=(const PPSDiamondG4Hit &right);
+  int operator==(const PPSDiamondG4Hit &) { return 0; }
+
+  void Draw() override {}
+  void Print() override;
+
+public:
+  G4ThreeVector getEntry() const;
```

Comment:
@mundim , in these 4 lines better to use G4ThreeVector& - by reference, no by value.

Reply:
Done.